test/built-ins/Temporal/PlainDate/calendar-invalid-iso-string.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2025 Igalia, S.L. All rights reserved.
```

Comment:
I think it's OK to put your own copyright here (yourself, or University of Bergen, whatever applies in this case.) Although the test is similar to one that Igalia wrote, but it's a new test.